README.rst

```diff
@@ -447,6 +447,105 @@ of the url, this will enable you to point all the API urls to
 your openwisp-network-topology API server's domain,
 example value: ``https://mytopology.myapp.com``.
 
+``API``
+-------
```

Comment:
please follow the same exact structure used in https://github.com/openwisp/openwisp-firmware-upgrader#rest-api

It must be very close, including the screenshots.